src/api/files.api.js

```diff
@@ -183,6 +183,45 @@ export const updateKml = (id, adminId, kml) => {
     })
 }
 
+/**
+ * Delete KML on backend
+ *
+ * @param {string} id KML ID
+ * @param {string} adminId KML admin ID
+ * @returns {Promise<void>}
+ */
+export const deleteKml = (id, adminId) => {
+    log.info('base url : ', kmlBaseUrl)
```

Comment:
This looks like a developement log which stayed in the code. You should remove this.